slices/cpio.yaml

```diff
@@ -0,0 +1,16 @@
+package: cpio
+
+essential:
+  - cpio_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/cpio:
+      /usr/bin/mt-gnu:
+  
+  copyright:
+    contents: 
```

Comment:
There is an extra whitespace character at the end of this `contents:` line, can you remove?